src/components/ShareMatchExperienceModal/index.tsx

```diff
@@ -51,11 +51,15 @@ const ShareMatchExperienceModal = ({ isOpen, onClose }: ShareMatchExperienceModa
         reset,
         getValues,
         trigger,
+        watch,
         formState: { errors },
     } = useForm<ShareMatchExperienceModalValues>({
         resolver: yupResolver(MatchExperienceSchema),
     });
 
+    const { matchDate, picture, ...rest } = watch();
```

Comment:
```suggestion
const { matchDate, picture, description, ...rest } = watch();
```
add that no need for desc in order to generate the desc it self. so it can give you head start for your desc writing